packages/manager/apps/carrier-sip/src/routing.js

```diff
@@ -12,6 +12,12 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => CarrierSipService.getCarrierSip(billingAccount, serviceName),
       cdrsLink: /* @ngInject */ ($state, billingAccount, serviceName) =>
         $state.href('app.cdrs', { billingAccount, serviceName }),
+      clusterDetails: /* @ngInject */ ($http, billingAccount, serviceName) =>
```

Comment:
Is that make sense for you to use a `catch` block?
To prevent blocking access to the dashboard I mean.